client/app/components/AboutUsTemplate/index.js

```diff
@@ -65,48 +65,42 @@ const AboutUsTemplate = () => (
           liUrl="https://www.linkedin.com/pub/garth-froese/b/863/324"
         />
         <ProfileCard
           name="Scott Rice"
           title="COO"
           imgSrc="http://res.cloudinary.com/dyjo5a3ci/image/upload/v1514087315/idicia/scott-rice.jpg"
-          description="Scott is the resident data nerd. Formerly a principal data scientist for one of the big 3 
-          credit bureaus. Scott writes, blogs, and remodels his house."
+          description="Scott is the resident data nerd. Formerly a principal data scientist for one of the big 3 credit bureaus. Scott writes, blogs, and remodels his house."
           liUrl="https://www.linkedin.com/pub/scott-rice/3/424/67"
         />
         <ProfileCard
           name="Tom Liebe"
           title="CTO"
           imgSrc="http://res.cloudinary.com/dyjo5a3ci/image/upload/v1514087315/idicia/tom-liebe.jpg"
-          description="As head of our technology department, he manages the team that builds and maintains our services. 
-          Tom enjoys reading, music and family time."
+          description="As head of our technology department, he manages the team that builds and maintains our services. Tom enjoys reading, music and family time."
           liUrl="https://ca.linkedin.com/pub/tom-liebe/8/a59/83"
-          />
+        />
         <ProfileCard
           name="Dave Mori"
           title="VP-BD"
           imgSrc="http://res.cloudinary.com/dyjo5a3ci/image/upload/v1514087315/idicia/dave-mori.jpg"
-          description="Self-proclaimed BBQ aficionado, judo practicioner, and father of 3 under 8. Dave is a great 
-          listener and can help you find your business solution."
+          description="Self-proclaimed BBQ aficionado, judo practicioner, and father of 3 under 8. Dave is a great listener and can help you find your business solution."
           liUrl="https://ca.linkedin.com/in/davemori"
         />
         <ProfileCard
           name="Ken Dick"
           title="CFO"
           imgSrc="http://res.cloudinary.com/dyjo5a3ci/image/upload/v1514087315/idicia/ken-dick.jpg"
-          description="Ken is our numbers guy and ensures everyone gets paid when he isn’t 
-          driving his convertible or motorcycle on roadtrips."
+          description="Ken is our numbers guy and ensures everyone gets paid when he isn't driving his convertible or motorcycle on roadtrips."
           liUrl="https://ca.linkedin.com/pub/ken-dick/69/830/410"
         />
         <ProfileCard
           name="Stacy Halligan"
           title="Carrier Rel."
           imgSrc="http://res.cloudinary.com/dyjo5a3ci/image/upload/v1514087315/idicia/stacey-halligan.jpg"
-          description="While Stacey is from Texas, don’t let the “y’all” fool you.  
-          Stacey is responsible for accessing our great telecommunications data."
+          description={`While Stacey is from Texas, don’t let the "y'all" fool you. Stacey is responsible for accessing our great telecommunications data.`}
```

Comment:
@leseanbiz let me know if this works or if the unique quote characters are necessary